src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -619,6 +663,40 @@ internal long SetRequestContent(HttpRequestMessage request, IDictionary content)
 
         }
 
+        internal void ParseLinkHeader(HttpResponseMessage response, System.Uri requestUri)
+        {
+            if (_relationLink == null)
+            {
+                _relationLink = new Dictionary<string, string>();
+            }
+            else
+            {
+                _relationLink.Clear();
+            }
+
+            // we only support the URL in angle brackets and `rel`, other attributes are ignored
+            // user can still parse it themselves via the Headers property
+            Regex regex = new Regex("<(?<url>.*?)>;\\srel=\"(?<rel>.*?)\"");
```

Comment:
You should not create a new `Regex` like this, instead you should prefer the static method, see the [best practices](https://msdn.microsoft.com/en-us/library/gg578045(v=vs.110).aspx).